src/server/worldserver/worldserver.conf.dist

```diff
@@ -3207,32 +3188,37 @@ MoveMaps.Enable = 1
 Minigob.Manabonk.Enable = 1
 
 #
-
 #    Allow.IP.Based.Action.Logging
 #        Description: Logs actions, e.g. account login and logout to name a few, based on IP of current session.
 #        Default:     0 - (Disabled)
 #                     1 - (Enabled)
+#
```

Comment:
Why add extra under? Shouldnt it only be one extra above?

Reply:
I don't know the standards and I always do this